README.md

```diff
@@ -5,3 +5,7 @@ This repo is for TiDB Challenge Program register and other relative events.
 ## Sign Up
 
 By [opening an issue](https://github.com/tidb-challenge-program/register/issues/new?template=challenge-program-register.md&title=Challenge+Program+Register), you can sign up challenge program. Once the sign up is successfully done, bot will comment on your issue.
+
+## Score Inheritance
```

Comment:
```suggestion
## Points Inheritance
```

Reply:
We use (Challenge) Points in readme and website. So I guess we should use it here too.